ckanext/hierarchy/plugin.py

```diff
@@ -64,6 +64,7 @@ def get_helpers(self):
                 'group_tree_get_longname': helpers.group_tree_get_longname,
                 'group_tree_highlight': helpers.group_tree_highlight,
                 'get_allowable_parent_groups': helpers.get_allowable_parent_groups,
+                'is_include_top_level_parent_group': helpers.is_include_top_level_parent_group,
```

Comment:
I think `include_top_level` is clearer.

Reply:
@amercader 
There is already `is_include_children_selected` helper. 
Also if we use `include_...` it could hide that we return a boolean.

Reply:
the existing helper means "Is `include_children` selected"?

Sorry but `is_include_top_level_parent_group` doesn't make much sense :)

Reply:
You're right